File: CS/Drill-Down Bootstrap Chart on JS/App_Code/BootstrapChartDataItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class BootstrapChartDataItem {
    public BootstrapChartDataItem(string fruitCategory, decimal categoryCount) {
        FruitCategory = fruitCategory;
        CategoryCount = categoryCount;
    }
    public string FruitCategory { get; set; }
    public decimal CategoryCount { get; set; }
}

File: VB/Drill-Down Bootstrap Chart on JS/App_Code/BootstrapChartDataItem.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class BootstrapChartDataItem
    Public Sub New(ByVal fruitCategory As String, ByVal categoryCount As Decimal)
        Me.FruitCategory = fruitCategory
        Me.CategoryCount = categoryCount
    End Sub
    Public Property FruitCategory() As String
    Public Property CategoryCount() As Decimal
End Class
